### ISSUE TITLE
Update to Swift 5.0

### ISSUE THREAD

**Issue by BigDada1** [ykyouhei/KYDrawerController#140]
After updating my app to Swift 5, I've got some (new) compiler warnings

- `KYDrawerController.swift:378:36: 'drawerController(_:stateChanged:)' is deprecated: renamed to 'drawerController(_:didChangeState:)'` (see #124)
- `KYDrawerController.swift:208:23: Expression implicitly coerced from 'UIView?' to 'Any'` (new)
- `KYDrawerController.swift:219:23: Expression implicitly coerced from 'UIView?' to 'Any'` (new)

**Comment by filipealva** [comment]
I know that it's still going to need to update to Swift 5.0 soon, but if you're stuck with these errors you can fix by adding this script at the end of your Podfile: 

```
post_install do |installer|
    installer.pods_project.targets.each do |target|
        if target.name == 'KYDrawerController'
            target.build_configurations.each do |config|
                config.build_settings['SWIFT_VERSION'] = '4.2'
            end
        end
    end
end
```

**Comment by BigDada1** [comment]
Yes, this Podfile script kills the `Expression implicitly coerced` warnings. But on the other side, there appears the `Conversion to Swift 5 is available` issue in the issue navigator – and #124 still remains in both variants.
